src/main/java/bio/terra/service/snapshot/flight/create/CreateSnapshotPrimaryDataQueryStep.java

```diff
@@ -1,112 +0,0 @@
-package bio.terra.service.snapshot.flight.create;
-
-import bio.terra.common.iam.AuthenticatedUserRequest;
-import bio.terra.grammar.Query;
-import bio.terra.grammar.exception.InvalidQueryException;
-import bio.terra.grammar.google.BigQueryVisitor;
-import bio.terra.model.DatasetModel;
-import bio.terra.model.SnapshotRequestModel;
-import bio.terra.model.SnapshotRequestQueryModel;
-import bio.terra.service.common.CommonFlightUtils;
-import bio.terra.service.dataset.AssetSpecification;
-import bio.terra.service.dataset.Dataset;
-import bio.terra.service.dataset.DatasetService;
-import bio.terra.service.snapshot.Snapshot;
-import bio.terra.service.snapshot.SnapshotDao;
-import bio.terra.service.snapshot.SnapshotService;
-import bio.terra.service.snapshot.exception.AssetNotFoundException;
-import bio.terra.service.tabulardata.google.bigquery.BigQuerySnapshotPdao;
-import bio.terra.stairway.FlightContext;
-import bio.terra.stairway.Step;
-import bio.terra.stairway.StepResult;
-import java.time.Instant;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-
-public class CreateSnapshotPrimaryDataQueryStep implements Step {
-
-  private final BigQuerySnapshotPdao bigQuerySnapshotPdao;
-  private final DatasetService datasetService;
-  private final SnapshotService snapshotService;
-  private final SnapshotDao snapshotDao;
-  private final SnapshotRequestModel snapshotReq;
-  private final AuthenticatedUserRequest userRequest;
-
-  public CreateSnapshotPrimaryDataQueryStep(
-      BigQuerySnapshotPdao bigQuerySnapshotPdao,
-      DatasetService datasetService,
-      SnapshotService snapshotService,
-      SnapshotDao snapshotDao,
-      SnapshotRequestModel snapshotReq,
-      AuthenticatedUserRequest userRequest) {
-    this.bigQuerySnapshotPdao = bigQuerySnapshotPdao;
-    this.datasetService = datasetService;
-    this.snapshotService = snapshotService;
-    this.snapshotDao = snapshotDao;
-    this.snapshotReq = snapshotReq;
-    this.userRequest = userRequest;
-  }
-
-  @Override
-  public StepResult doStep(FlightContext context) throws InterruptedException {
```

Comment:
Note: This is the original GCP step that we're now sharing the logic between GCP & Azure.